packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -322,15 +322,16 @@ async function unzipApp (zipPath, dstRoot, supportedAppExtensions) {
   try {
     logger.debug(`Unzipping '${zipPath}'`);
     const timer = new timing.Timer().start();
+    const useSystemUnzipEnv = process.env.APPIUM_PREFER_SYSTEM_UNZIP;
+    const useSystemUnzip = _.isEmpty(useSystemUnzipEnv)
+      || !['0', 'false'].includes(_.toLower(useSystemUnzipEnv));
     /**
      * Attempt to use use the system `unzip` (e.g., `/usr/bin/unzip`) due
      * to the significant performance improvement it provides over the native
      * JS "unzip" implementation.
      * @type {import('@appium/support/lib/zip').ExtractAllOptions}
      */
-    const extractionOpts = {
-      useSystemUnzip: !system.isWindows(),
```

Comment:
was `@appium/support` module version bumped to use this functionality?

Reply:
yea. the package.json already has the latest `@appium/support` package.